packages/cli-repl/src/cli-repl.ts

```diff
@@ -340,7 +389,41 @@ class CliRepl {
     if (hasMisspelledFilename) {
       const msg =
         'Warning: Found ~/.mongoshrc, but not ~/.mongoshrc.js. Did you forget to add .js?\n';
-      this.output.write(this.clr(msg, ['bold', 'yellow']));
+      this.output.write(this.clr(msg, 'mongosh:warning'));
+    }
+  }
+
+  async loadGlobalConfigFile(): Promise<Partial<CliUserConfig>> {
+    let fileContents = '';
+    let filename = '';
+    for (filename of this.globalConfigPaths) {
+      try {
+        fileContents = await fs.readFile(filename, 'utf8');
+        break;
+      } catch { /* ignore error */ }
```

Comment:
I'm wondering if all errors should be ignored. Like.. what if one of them exists but there's some permissions error - should it not warn in that case? Maybe too much of an edge case to worry about.

Reply:
Right, I also wondered about that because we do handle this a bit better for the user’s home directory files. Printing a warning might be annoying though, because users would frequently not be able to _act_ on it.

I’ve pushed an update to make it so that we log errors other than ENOENT. I think that’s reasonable as a debugging mechanism if this does happen.